### PR TITLE
[DC-1232] Assess Safety of Removing EHR Site Validation File Writer Endpoint

### PR DESCRIPTION
**Pull request by dcampbell-vumc**
* removed code creating /ParticipantValidation/SiteFiles endpoint handling in `/validation/main.py`
* removed `WRITE_SITE_VALIDATION_FILES` constant from `constants/validation/main.py`
* removed `write_sites-pii_validation_files()` function from `/validation/main.py`

### CODE REVIEW

File: data_steward/validation/main.py [hunk 1]
```diff
@@ -1001,17 +1001,6 @@ def write_drc_pii_validation_file():
...
 
-@api_util.auth_required_cron
-@log_traceback
-def write_sites_pii_validation_files():
-    project = bq_utils.app_identity.get_application_id()
-    validation_dataset = bq_utils.get_validation_results_dataset_id()
-    logging.info(f"Calling write_results_to_site_buckets")
-    matching.write_results_to_site_buckets(project, validation_dataset)
-
-    return consts.SITES_VALIDATION_REPORT_SUCCESS
```
Comment (lrwb-aou) [inline data_steward/validation/main.py:1012]: Need to remove this from the constants file too.

**Review by lrwb-aou — COMMENTED** [review]
Please make the one additional removal.

File: data_steward/validation/main.py (hunk 1 above)
Comment (dcampbell-vumc) [reply data_steward/validation/main.py:1012]: Removed!

**Review by lrwb-aou — APPROVED** [review]
Thank you!

**Pull request merged**